data/macos.yaml

```diff
@@ -11,6 +11,21 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#System_Logs'
 ---
+name: MacOSAppleUnifiedLog
```

Comment:
please rename to "MacOSUnifiedLogging", since this is the name Apple is using.

https://developer.apple.com/videos/play/wwdc2016/721/
https://developer.apple.com/documentation/os/logging?language=occ

Reply:
fixed